instrumentation/opentelemetry-instrumentation-sqlalchemy/src/opentelemetry/instrumentation/sqlalchemy/__init__.py

```diff
@@ -76,6 +89,13 @@ def _instrument(self, **kwargs):
         """
         _w("sqlalchemy", "create_engine", _wrap_create_engine)
         _w("sqlalchemy.engine", "create_engine", _wrap_create_engine)
+        if sqlalchemy.__version__.startswith("1.4"):
```

Comment:
Would this only work for version 1.4, what happens with version 1.5 is released? Would it make sense to use [packaging](https://packaging.pypa.io/en/latest/version.html#packaging.version.parse) to compare the version?

Reply:
Good point - I've updated it to use packaging.